programs-remote-tests/tests/acceptance/shortcodesCest.php

```diff
@@ -2,19 +2,79 @@
 
 class shortcodesCest
 {
-    public function listEvents(AcceptanceTester $I)
+    public function listEventsByCategory(AcceptanceTester $I)
     {
         $I->amOnPage('/shortcode-event-list');
         $I->see('Example Program');
+        $I->see('Exhaustive Program');
+
+        $I->dontSee('Lodging Hotel Program');
+        $I->dontSee('Lodging Price Program');
+        $I->dontSee('Renter Program');
+        $I->dontSee('Multi Person Lodging');
+        $I->dontSee('Multi Person Tiered');
     }
 
     public function listEventsTable(AcceptanceTester $I)
     {
         $I->amOnPage('/shortcode-event-list-table');
-        $I->see('Available Spots');
-        $I->see('Register');
-        $I->see('Events');
-        $I->see('Dates');
+
+        $I->wantTo('Verify that the headers are correct');
+        $I->see('Dates', '.rs-program thead .rs-dates');
+        $I->see('Events', '.rs-program thead .rs-title');
+        $I->see('Price from', '.rs-program thead .rs-price-first');
+        $I->see('Details', '.rs-program thead .rs-show-more-link');
+        $I->see('Available Spots', '.rs-program thead .rs-availability');
+        $I->see('Register', '.rs-program thead .rs-show-register-link');
+
+        $I->see('Flexible Dates', 'tbody tr:nth-child(1) .rs-dates');
+        $I->see('Lodging Hotel Program', 'tbody tr:nth-child(1) .rs-title');
+        $I->see('View Details', 'tbody tr:nth-child(1) .rs-show-more-link');
+        $I->see('Register Now', 'tbody tr:nth-child(1) .rs-show-register-link');
+    }
+
+    public function eventListHideDescription(AcceptanceTester $I)
+    {
+        $I->amOnPage('/shortcode-event-list-hide-text/');
+
+        $I->see('Exhaustive Program');
+        $I->see('No Price Program');
+
+        $I->wantTo('Verify that I can\'t see description');
+        $I->dontSee('Mauris placerat eleifend leo. Quisque sit amet est et sapien ullamcorper pharetra');
```

Comment:
Using this latin in tests seems questionable. It looks auto-generated even though it isn't. I'd rather if it said:
$I->dontSee('first few works of exhaustive program description.');
But I realize that would take more time to do. (though we could push the changes to production directly). 
Or is there another program we can use with english description?